Asgardia.Constitution.Proposal.txt

```diff
@@ -345,75 +339,120 @@ Article 31. Legal System and Branch structures
--- While in Office, the Head of State shall be referred only by titles of “Asgardian Head of State” or “Head of Asgardia.” No suggestion of Noble title may be used.
-7. In the event the Head of State dies, resigns, becomes permanently incapacitated or is declared missing, as well as in the circumstances stipulated by law where the Head of State is unable to perform his/her duties, his/her duties temporarily pass to the Chairperson of the Council of Supreme Values until the reasons which prevented the Head of State from performing such duties cease to be in effect or until the inauguration of the new Head of State. The Head of State must prove his fitness within two Earth-months and by the approval of a certified medical physician duly appointed by the Ministry of Science. In the event that both the Head of State and Chairperson of Council of Supreme Values are unable to perform such duties define herein for the Head of State, the responsibilities shall fall to the following line of succession:
- - Ministry of Science Chairperson
- - Ministry of Safety and Security Chairperson
- - Ministry of Finance Chairperson
- - Ministry of Administrative Affairs Chairperson
- - Ministry of Justice Chairperson
--- The Head of State:
- a. appoints the Supreme Justice of the Judicial Court, the Chairperson of the National Audit Office, the Chairperson of the Council of Supreme Values and Asgardian ambassadors extraordinary and plenipotentiary to Earth nations;
- b. has the right to veto candidates nominated in accordance with the Constitution and the Law for the positions of Chairperson of the Government, member of the Council of Supreme Values, Chairman of the National Bank, and has the right to propose removal of persons who hold these positions;
+§ 1. The Head of State is limited to a maximum of two terms. Each term shall not exceed 5 years of civic service.
+§ 2. The HoS shall be a minimum age of 35, hold a professional certification in law and/or science, and be a citizen of Asgardia for at least five years. The required merits of Candidates for HoS shall be further defined by law.
+§ 3. The Head of State is not above the law of Asgardia.
+§ 4. The Head of Nation role shall be abolished upon the adoption of this Constitution. The current Head of Nation, as defined by Decree # 1, shall be requested to take the honorable role of Head of State. After five years, the first Head of State shall agree to step down unless voted to a second term in office. In the event the Head of Nation declines to become the Head of State, up to five Candidates shall be vetted by special committee appointed by the Head of Nation, within 30 Earth-days, and hold a general Election where the candidate with the most votes shall become the Head of State. In the case where two or more Candidates have the same number of Votes, the Head of Nation shall cast a tie-breaker vote.

```

Comment:
Should this section be moved to Chapter 10 as this is really part of the transition out of the formation phase?